hardhat.config.js

```diff
@@ -1,47 +1,49 @@
 require('@nomicfoundation/hardhat-toolbox');
+require('@nomiclabs/hardhat-ethers');
+require('@typechain/hardhat');
 
-const env = process.env.NETWORK || 'testnet';
-const { importNetworks, readJSON } = require(`${__dirname}/axelar-chains-config`);
-const chains = require(`${__dirname}/axelar-chains-config/info/${env}.json`);
-const keys = readJSON(`${__dirname}/keys.json`);
+const { importNetworks, readJSON } = require('./axelar-chains-config');
```

Comment:
```suggestion
const { importNetworks, readJSON } = require(`${__dirname}/axelar-chains-config`);
```
avoid relative paths, so it doesn't matter which folder the user runs the script from